src/fn/axial.ts

```diff
@@ -11,7 +11,7 @@ import { silkscreenRef, type SilkscreenRef } from "../helpers/silkscreenRef"
 
 export const axial_def = z.object({
   fn: z.string(),
-  num_pins: z.any().transform(() => 2),
+  num_pins: z.literal(2).default(2),
```

Comment:
Hmm i think this needs optional as well?

Reply:
I think we need it to be read only so we don't have to add optional. and literal without default gives me errors

Reply:
Ah ok sounds good